.azure-pipelines/linux_build.yml

```diff
@@ -0,0 +1,56 @@
+steps:
+- bash: |
+    source ${CONDA}/etc/profile.d/conda.sh
+    sudo chown -R ${USER} ${CONDA}
+    conda config --set always_yes yes --set changeps1 no
+    conda update -q conda
+    conda info -a
+    conda create --name rdkit_build $(compiler) cmake \
+        boost-cpp=$(boost_version) boost=$(boost_version) \
+        py-boost=$(boost_version) libboost=$(boost_version) \
+        numpy matplotlib cairo pillow eigen pandas
+  displayName: Setup build environment
+- bash: |
+    source ${CONDA}/etc/profile.d/conda.sh
+    conda activate rdkit_build
+    mkdir build && cd build && \
+    cmake .. \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DRDK_INSTALL_INTREE=ON \
+    -DRDK_INSTALL_STATIC_LIBS=OFF \
+    -DRDK_BUILD_CPP_TESTS=ON \
+    -DRDK_BUILD_PYTHON_WRAPPERS=ON \
+    -DRDK_USE_BOOST_REGEX=ON \
+    -DRDK_BUILD_COORDGEN_SUPPORT=ON \
+    -DRDK_OPTIMIZE_NATIVE=ON \
+    -DRDK_BUILD_TEST_GZIP=ON \
+    -DRDK_BUILD_FREESASA_SUPPORT=ON \
+    -DRDK_BUILD_AVALON_SUPPORT=ON \
+    -DRDK_BUILD_INCHI_SUPPORT=ON \
+    -DRDK_BUILD_SWIG_WRAPPERS=OFF \
+    -DRDK_SWIG_STATIC=OFF \
+    -DRDK_BUILD_THREADSAFE_SSS=ON \
+    -DRDK_TEST_MULTITHREADED=ON \
+    -DBoost_NO_SYSTEM_PATHS=ON \
+    -DRDK_BOOST_PYTHON3_NAME=$(python_name) \
+    -DPYTHON_EXECUTABLE=${CONDA_PREFIX}/bin/python3 \
+    -DCMAKE_INCLUDE_PATH="${CONDA_PREFIX}/include" \
+    -DCMAKE_LIBRARY_PATH="${CONDA_PREFIX}/lib"
+  displayName: Configure build (Run CMake)
+- bash: |
+    source ${CONDA}/etc/profile.d/conda.sh
+    conda activate rdkit_build
+    cd build
+    make install
```

Comment:
Probably should have `-j $( $(number_of_cores) )` (or whatever the appropriate spelling of that is here) so that the build runs in parallel.

Reply:
I tried that, but the build failed due to the VM running out of memory. But I'll try again, just to make sure.

Reply:
Looks like it works. I enabled parallel builds for linux.